client/src/pages/private/ParticipantTable.js

```diff
@@ -302,34 +241,7 @@ export default () => {
 
         const index = rows.findIndex(row => row.id === participantId);
         const { firstName, lastName } = rows[index];
-
-        const toasts = {
-          open: {
-            status: ToastStatus.Info,
-            message: `${firstName} ${lastName} is has been disengaged`,
-          },
-          interviewing: {
-            status: ToastStatus.Info,
-            message: `${firstName} ${lastName} is now being interviewed`,
-          },
-          offer_made: {
-            status: ToastStatus.Info,
-            message: `${firstName} ${lastName} has been made a job offer`,
-          },
-          hired: {
-            status: ToastStatus.Success,
-            message: `${firstName} ${lastName} has been hired`,
-          },
-          rejected: {
-            status: ToastStatus.Info,
-            message: `${firstName} ${lastName} has been archived`,
-          },
-          already_hired: {
-            status: ToastStatus.Info,
-            message: `${firstName} ${lastName} is already hired by someone else`,
-          }
-        };
-
+        const toasts = makeToasts(firstName,lastName,ToastStatus);
```

Comment:
Do we need to import ToastStatus here? Can we import it in the constants doc and remove it as a parameter in the makeToasts function?